cmd/yorkie/version.go

```diff
@@ -19,26 +19,43 @@ package main
 import (
 	"fmt"
 	"runtime"
+	"runtime/debug"
 
 	"github.com/spf13/cobra"
 
 	"github.com/yorkie-team/yorkie/internal/version"
 )
 
+const commitRevisionKey = "vcs.revision"
+const buildTimeKey = "vcs.time"
```

Comment:
`vcs.time` might be commit time, not build time. I think it would be better to keep the existing build time. What do you think? 🤔

https://cs.opensource.google/go/go/+/refs/tags/go1.18.5:src/cmd/go/internal/vcs/vcs.go;l=60

Reply:
It may not fit as much as the time of vcs(commit). However, I thought it would be consistent in the main-based release environment.
This information is important in most deployment environments.